rpc/gnmi/gnmi-specification.md

```diff
@@ -922,10 +1005,15 @@ it desires.
 
 A target receiving a `SetRequest` message processes the operations specified
 within it - which are treated as a transaction (see [Section
-3.4.3](#343-transactions)). The server MUST process deleted paths (within the
-`delete` field of the `SetRequest`), followed by paths to be replaced (within
-the `replace` field), and finally updated paths (within the `update` field). The
-order of the replace and update fields MUST be treated as significant within a
+3.4.3](#343-transactions)). The server MUST process paths in the following order:
+
+1. deleted paths (within the`delete` field of the `SetRequest`)
+2. replaced paths (within the `replace` field)
+3. updated paths (within the `update` field)
+or as a single set of union_replace operations.  That is, union_replace is not
+mixed with other operations in a SetRequest.
```

Comment:
This wording isn't that clear to me. I think it's better to just say.

> Where a `SetRequest` contains `delete`, `replace`, and `update` fields, the server MUST process paths in the following order:
> 1. deleted paths...
> 2. replaced paths...
> 3. updated paths...
>
> A `SetRequest` containing `delete`, `replace` and `update` MUST NOT contain `union_replace` operations.

Reply:
Here, a forward reference to what `union_replace` is would be useful (just a link to the relevant section).

Reply:
Done